spec/index.html

```diff
@@ -324,39 +325,42 @@ <h3>Shared blank nodes, unions and merges</h3>
       to identify a blank node, but unless these documents are in a shared identifier scope
       or are derived from a common source,
       the occurrences of "<code>_:x</code>" in one document
-      will identify a different blank node than the one in the graph described by the other document.
+      will identify a different blank node than the occurrences of "<code>_:x</code>"
+      in the graph described by the other document.
       When graphs are formed by combining RDF from multiple sources,
-      it may be necessary to <dfn class="no-export lint-ignore">standardize</dfn> apart the blank node identifiers
-      by replacing them by others which do not occur in the other document(s).
-      For example, the two graphs represented  by the following texts:</p>
+      it may be necessary to <dfn class="no-export lint-ignore">standardize</dfn> the blank node identifiers
```

Comment:
"standardization apart" is a fixed term (Logic Programming) and I would therefore not change that

Reply:
RDF Semantics is not defining "standardize apart".

Just remove the <dfn> to leave the text: 

"it may be necessary to standardize apart the blank node identifiers"

Reply:
I think @afs advises well (hence part of this text suggestion), but based on what I found in documentation [Google showed me when I asked "what does "Standardize apart" mean?"](https://www.google.com/search?q=what+does+%22Standardize+apart%22+mean%3F), I've reordered some words, which I think makes the sentence more readable. (We might consider adding an external link to some Logic Programming text which goes further into this subject, which I believe we have not made a prerequisite for readers of our specifications.)
```suggestion
      it may be necessary to standardize the blank node identifiers apart,
```

Reply:
The operation name is "standardize apart" as @doerthe notes.
The google hits has it as the two words together each time.

For better or worse, the RDF Semantics introduction starts as the document goes on:
"This document defines a model-theoretic semantics ..."

(and why I haven't done a technical review of RDF Semantics.)

Reply:
Any objection to *this*, which makes plain that it's a complex verb? I have spent some fruitless time searching for a page that clearly and simply defines this "operation", but all pages I've found treat it as preexisting knowledge, which is clearly a problematic assumption, as it was not preexisting for me. I would much rather entirely remove this term of art, and say the same thing with other words.
```suggestion
      it may be necessary to "standardize apart" the blank node identifiers
```

Reply:
I think it is OK but (1) I am not an editor of this document (2) experts in the field may have a view.

Reply:
> _ping_ oh ye experts in the field (@doerthe @franconi @niklasl (are there other such experts among us?))! Please chime in on [my latest suggested change](https://github.com/w3c/rdf-semantics/pull/126#discussion_r2021363646)?

I think it is ok, as it uses the appropriate term.

Reply:
Why not `<dfn class="no-export lint-ignore">standardize</dfn>` ?. There is a link to that later in the document. If we remove the dfn context here, we also need to remove the link (line 361). Alternatively, the dfn could be for "standardize apart"

Reply:
> [@doerthe] Alternatively, the `dfn` could be for "standardize apart"

I could go for that. In fact, [I did on March 28](https://github.com/w3c/rdf-semantics/pull/126#discussion_r2019116689).

> [@afs] RDF Semantics is not defining "standardize apart".

The `dfn` tag does not necessarily indicate that this document is defining a term for use in other documents. It can and often does indicate the location of the working definition being used within this document, which linkage is useful for readers unfamiliar with such niche terminology as "standardize apart".

Reply:
You can use `dfn` with `data-cite` which will style and reference the term, but not define it within the document. Clicking on it would take you to the `data-cite` reference, which must be of the form `SPECREF#anchor`. But, not strictly necessary if people reading this would understand the terminology without an explicit reference.